bundle/manifests/operator-certification-operator.clusterserviceversion.yaml

```diff
@@ -340,4 +426,4 @@ spec:
   minKubeVersion: 1.21.0
   provider:
     name: Red Hat
-  version: 0.0.0
+  version: 0.0.1
```

Comment:
Since the container image will be of a `1.0.0` release we should update this version as well.